Makefile.build

```diff
@@ -630,15 +627,65 @@ test-suite: world byte $(ALLSTDLIB).v
 # Default rules for compiling ML code
 ###########################################################################
 
-# Target for libraries .cma and .cmxa
+gramlib__pack:
+	mkdir -p $@
+
+# gramlib.ml contents
+gramlib__pack/gramlib.ml: gramlib__pack
```

Comment:
I think this should be an order only dependency, because touching a file in a directory updates the directory's timestamp it would cause a rebuild everytime otherwise.

Reply:
Care to explain how to do this?

Reply:
put a pipe before it: `.../gramlib.ml: | gramlib__pack`